### PR TITLE
Ft/footer

### PR DESCRIPTION
**Pull request by albert-byiringiro**
# PR Description

This includes a responsive footer and a separate section for the red figure that comes right before the footer on some pages

## Description of tasks that were expected to be completed

List all the tasks and sub-tasks that were assigned to you Make sure you check every completed task. list them in this style

- [x] Create a footer section and make it responsive
- [x] Create a sticky-red section and make it responsive
- [x] Add both sticky-red and footer on the index page 

## Github board link (If appropriate/applicable)

Provide github board link related to your PR in case it's appropriate and applicable

## Screenshots (If appropriate/applicable)
Provide screenshots related to your PR in case it's appropriate and applicable

## Please check this Checklist before you submit your PR:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My code generates no warnings
- [x] There are no vulnerabilities

### CODE REVIEW

File: src/sections/footer.html [hunk 1]
```diff
@@ -1,11 +1,34 @@
...
+        <li class="md:order-2 lg:w-[27%]">
+          <a href="" class="">Impressum</a>
+        </li>
+        <li class= lg:w-[27%]">
+          <a href="" class="">LinkedIn</a>
+        </li>
+        <br class="hidden lg:block" />
+        <li class= lg:w-[27%]">
+          <a href="" class="">Instagram</a>
+        </li>
```
Comment (kareraolivier) [inline src/sections/footer.html:31]: Map this list

**Review by kareraolivier — COMMENTED** [review]
This is great
You only have one optional comment

File: src/sections/footer.html (hunk 1 above)
Comment (InshutiSouvede) [reply src/sections/footer.html]: we have worked on this now

**Review by kareraolivier — COMMENTED** [review]
Well done I left small thing to look on

File: src/sections/footer.html [hunk 2]
```diff
@@ -1,11 +1,34 @@
...
+        <li class="md:order-2 lg:w-[27%]">
+          <a href="" class="">Impressum</a>
+        </li>
+        <li class= lg:w-[27%]">
+          <a href="" class="">LinkedIn</a>
+        </li>
+         <br class="hidden lg:block" /> 
+        <li class= lg:w-[27%]">
+          <a href="" class="">Instagram</a>
+        </li> -->
```
[the remark this answers is not in the document]
Comment (kareraolivier) [reply src/sections/footer.html:31]: good work remove the commented data

[the remark this answers is not in the document]
Comment (InshutiSouvede) [reply src/sections/footer.html]: I did it now

**Pull request merged**